Exercises-2-Basic/src/exercises/Num5.java

```diff
@@ -0,0 +1,30 @@
+package exercises;
+
+import java.util.Scanner;
+
+public class Num5 {
+    /*
+     * 5. Write a java program that compares two given numbers, and prints the
+     * relation between them >, < , =. For example, Input: 5 , 10 Your program
+     * should print "5 < 10"
+     */
+    public static void main(String[] args) {
```

Comment:
The solution is not working properly.
See what happens when you enter equal numbers.

Reply:
Done